.github/workflows/cmake.yml

```diff
@@ -1,86 +0,0 @@
-name: CMake
-
-on: [push, pull_request]
-
-jobs:
-  build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      # Don't cancel the macOS build if the Linux build fails, etc.
-      fail-fast: false
-      matrix:
-        os: [ubuntu-20.04, macos-11, windows-2019]
-        use-asan: [YES, NO]
-        exclude:
-          - os: windows-2019
-            use-asan: YES
-    env:
-        # Expose to the build-*.sh in an environment variable
-        USE_ASAN: ${{ matrix.use-asan }}
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        submodules: 'recursive'
-
-    # https://github.com/ilammy/msvc-dev-cmd
-    - name: Setup MSVC environment
-      if: runner.os == 'Windows' 
-      uses: ilammy/msvc-dev-cmd@v1
-
-    - name: Linux Build
```

Comment:
I introduced the following naming convention for better understanding: `[Operating System]: [Concise title of the step]`